mdal/api/mdal.h

```diff
@@ -347,18 +349,20 @@ MDAL_EXPORT int MDAL_D_valueCount( DatasetH dataset );
 //! Returns whether dataset is valid
 MDAL_EXPORT bool MDAL_D_isValid( DatasetH dataset );
 
+//! Returns whether dataset supports active flag for dataset faces
+MDAL_EXPORT bool MDAL_D_hasActiveFlagCapability( DatasetH dataset );
```

Comment:
Maybe it would be better to use a mix of capability flags... e.g. `int MDAL_D_capabilities(dataset)` - and each capability would be a different bit? We would not need to add extra exported methods to the API...

But I am easy - we can also leave it as is...

Reply:
we have this version, also for drivers etc, see `MDAL_DR_saveMeshCapability` . lets keep it this way (Consistent) we can always change in the future